.cproject

```diff
@@ -1,266 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Did you intend to delete the .cproject and .project files? This isn't well maintained and was added for one of the IDE examples (OpenSTM32 I think).

Reply:
These are Eclipse project files and having these files clashes with Eclipse project settings (always having to remember to omit these files during commits and backing up these files in case of jumping between git branches). Perhaps it would be better to rename them something like`OpenSTM32.cproject` and `OpenSTM32.project`? This way setting up a custom Eclipse project will be easier.

Reply:
Yes go ahead and rename those instead of deleting them and add a line in the IDE/OPENSTM32/README.md indicating.

Also please rebase this branch to latest master and force push as it has a conflict on wolfcrypt/src/include.am. Or do a merge with master to resolve.

Reply:
According to `git log`, these files were added with LPCXpresso IDE support and the note about the file name change should occur in `IDE/LPCXPRESSO/README.md`. Could you confirm?

Reply:
Thanks for checking. Confirmed... I added these with LPCExpresso `IDE/LPCXPRESSO/README.md`.

Reply:
Please check if the additional instruction in `IDE/LPCXPRESSO/README.md` is sufficient.